src/graph/ShowExecutor.cpp

```diff
@@ -1180,6 +1183,54 @@ std::string ShowExecutor::roleToStr(nebula::cpp2::RoleType type) {
     return role;
 }
 
+void ShowExecutor::showSessions() {
+    auto future = ectx()->getMetaClient()->listSessions();
+    auto *runner = ectx()->rctx()->runner();
+
+    auto cb = [this] (auto &&resp) {
+        if (!resp.ok()) {
+            doError(std::move(resp).status());
+            return;
+        }
+
+        auto sessionItems = std::move(resp).value();
+        resp_ = std::make_unique<cpp2::ExecutionResponse>();
+        std::vector<cpp2::RowValue> rows;
+        std::vector<std::string> header{"Host:Port", "Session ID", "Start Time", "Update Time"};
+        resp_->set_column_names(std::move(header));
+
+        std::sort(sessionItems.begin(), sessionItems.end(), [] (const auto& a, const auto& b) {
+            // sort with addr and ip
+            if (a.get_addr() == b.get_addr()) {
+                return a.get_session_id() < b.get_session_id();
+            }
+            return a.get_addr() < b.get_addr();
+        });
+
+        for (auto& item : sessionItems) {
+            std::vector<cpp2::ColumnValue> row;
+            row.resize(4);
+            row[0].set_str(item.get_addr());
+            row[1].set_integer(item.get_session_id());
+            row[2].set_integer(item.get_start_time());
```

Comment:
it‘s better to show by data time format, wdyt

Reply:
Good point,  👍